# The incoming info message from old coalition leader to new coalition leader

uint64 sendingTime #The sending time

uint16 infoTypeID

string infoMessage
